### PR TITLE
GLAD-227

### PR DESCRIPTION
**Pull request by sourabh0332**

### CODE REVIEW

File: public/plugin/js/app/fullview/applozic.fullview.js [hunk 1]
```diff
@@ -8100,8 +8100,10 @@ var MCK_CLIENT_GROUP_MAP = [];
                                     $mck_tab_status.removeClass('vis').addClass('n-vis');
                                     $mck_typing_label.html(MCK_LABELS['typing']);
                                 }
-                                $mck_typing_box.removeClass('n-vis').addClass('vis');
-                                setTimeout(function() {
+                                var userBlockStatus=document.getElementById('mck-block-button').title;
+                                if(userBlockStatus == "Block User"){
+                                    $mck_typing_box.removeClass('n-vis').addClass('vis');
+                                }setTimeout(function() {
```
Comment (devashishmamgain) [inline public/plugin/js/app/fullview/applozic.fullview.js]: @sourabh0332 move setTimeout to next line.

Comment (sourabh0332) [reply public/plugin/js/app/fullview/applozic.fullview.js]: sir is i revert or not
its glad-320

Comment (devashishmamgain) [reply public/plugin/js/app/fullview/applozic.fullview.js]: no revert, just move it to next line and commit in the same branch, that will get reflected here.

**Comment by adarshmishra** [comment]
closing this, no need to merge.

**Pull request closed without merging**